src/webserver/Static.jl

```diff
@@ -334,4 +334,4 @@ function http_router_for(session::ServerSession)
     HTTP.@register(router, "GET", "/favicon.ico", create_serve_onefile(project_relative_path(frontend_directory(allow_bundled=false), "img", "favicon.ico")))
 
     return router
-end
\ No newline at end of file
+end
```

Comment:
Thanks for this!

Reply:
wasn't me! i dont care of course :)

Reply:
It's sooooo annoying when GitHub places the ⛔️